go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/sigstore/cosign
+module github.com/sigstore/cosign/v2
 
 go 1.18
```

Comment:
should we bump this to 1.19?  1.20 is almost out and 1.18 will be deprecated

Reply:
I think that's smart but let's do it in a different PR